chrome/content/zotero/xpcom/citeproc.js

```diff
@@ -14388,12 +14373,78 @@ CSL.Engine.prototype.retrieveAllStyleModules = function (jurisdictionList) {
 }
 CSL.parseParticles = function(){
     var PARTICLES = [
+        ["al-", [[[0,1], null],[null,[0,1]]]],
+        ["at-", [[[0,1], null],[null,[0,1]]]],
+        ["ath-", [[[0,1], null],[null,[0,1]]]],
+        ["aá¹¯-", [[[0,1], null],[null,[0,1]]]],
```

Comment:
Yeah. Looks like bitbucket serves this incorrectly in the Raw view. Will fix later today